server/src/index.ts

```diff
@@ -5,6 +5,8 @@
 dotenv.config()
 const app = express()
 import { getConnection } from './dao/connection'
+import { logger } from './util/logger'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/c0dejohn/e-commerce-backend/pullRequest?prid=8292833)
```suggestion
import { logger } from "./util/logger"
```